### ISSUE TITLE
[FavoriteMedia]: Fav media tabs no longer showing

### ISSUE THREAD

**Issue by bob-carter** [Dastan21/BDAddons#249]
### Discord Build

Stable

### Discord Version

stable 1.10.1

### Plugin Version

v1.9.0

### Steps To Reproduce

Install DiscordCanary, BD, then setup plugin
Click images button.

### Describe the bug

It's just not showing up

### Screenshots

![image](https://github.com/Dastan21/BDAddons/assets/46971019/a466661d-9044-44ee-9257-e2395f3dcae5)

**Comment by vt-idiot** [comment]
Same issue on ptb
```md
## Discord Info
ptb 288684 (4ec6567) Host 1.0.1068 x86 (47107) Windows 10 64-bit (10.0.19045)

## BetterDiscord
stable 1.10.1
```

**Comment by TheWolfCraft1** [comment]
I have the same issue, but it's not going away. I've attached a log file in the hope someone can find a solution
[discord.com-1715999300163.log](https://github.com/Dastan21/BDAddons/files/15359411/discord.com-1715999300163.log)

**Comment by Dastan21** [comment]
Fixed in v1.9.1, waiting for BD approval.
